exchanges/account/account_types.go

```diff
@@ -59,6 +60,7 @@ type Balance struct {
 	Free                   float64
 	AvailableWithoutBorrow float64
 	Borrowed               float64
+	UpdatedAt              time.Time
```

Comment:
Main thing I'd like to request of you is to update the rpc server definitions to include this.
in `/gctrpc/rpc.proto` update `message AccountCurrencyInfo {` to have `UpdatedAt`, regenerate proto definitions via `/gctrpc/buf generate`